move some methods out of results/trim them (#434)
AlexsLemonade/OpenPBTA-manuscript: content/03.results.md

### Crowd-sourced Somatic Analyses to Create an Open Pediatric Brain Tumor Atlas

We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines [@doi:10.1093/neuonc/noz192] from 943 patients from the Pediatric Brain Tumor Atlas (PBTA), consisting of 911 patients from the [Children’s Brain Tumor Network (CBTN)](https://CBTN.org) [@doi:10.1016/j.neo.2022.100846] and 32 patients from the [Pacific Pediatric Neuro-Oncology Consortium (PNOC)](https://pnoc.us/) [@doi:10.1002/ijc.32258; @doi:10.1158/1078-0432.CCR-22-0803] (**Figure {@fig:Fig1}A**).

> A summarized view of the number of biospecimens per phase of therapy across broad histologies and cancer groups is shown in (**Figure {@fig:Fig1}B**). We then harnessed, and built upon, the benchmarking efforts of the [Gabriella Miller Kids First Data Resource Center](https://kidsfirstdrc.org/) to develop robust and reproducible data analysis workflows within the [CAVATICA platform](https://www.cavatica.org/) to perform primary somatic analyses including calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and gene fusions (**Figure {@fig:S1}**) and **STAR Methods**). 

Reviewer: more word cuts - 
```suggestion
**Figure {@fig:Fig1}B** summarizes the number of biospecimens per phase of therapy across broad histologies and cancer groups. 
We harnessed, and built upon, the benchmarking efforts of the [Gabriella Miller Kids First Data Resource Center](https://kidsfirstdrc.org/) to develop robust and reproducible data analysis workflows within the [CAVATICA platform](https://www.cavatica.org/) to perform primary somatic analyses including calling single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and gene fusions (**Figure {@fig:S1}**) and **STAR Methods**). 
```

We created a public Github analysis repository ([https://github.com/AlexsLemonade/OpenPBTA-analysis](https://github.com/AlexsLemonade/OpenPBTA-analysis)) to hold all code associated with analyses downstream of the Kids First workflows and a GitHub manuscript repository ([https://github.com/AlexsLemonade/OpenPBTA-manuscript](https://github.com/AlexsLemonade/OpenPBTA-manuscript)) with Manubot [@doi:10.1371/journal.pcbi.1007128] integration to enable real-time manuscript creation.
With very few exceptions noted in their respective analysis module READMEs, most modules can be run locally, and since they are run in the project Docker&reg; [@https://dl.acm.org/doi/10.5555/2600239.2600241] container, they can easily be scaled and/or used on an Amazon EC2 instance.
Importantly, all analyses and manuscript writing were conducted openly throughout the research project, allowing any researcher in the world the opportunity to contribute.

The process for analysis and manuscript contributions is outlined in **Figure {@fig:Fig1}C**.
First, a potential contributor would propose an analysis by filing an issue in the GitHub analysis repository.

> Next, organizers for the project, or other contributors with expertise, provide feedback about the proposed analysis (**Figure {@fig:Fig1}C**).

Reviewer: ```suggestion
Next, organizers for the project, or other contributors with expertise, provided feedback about the proposed analysis (**Figure {@fig:Fig1}C**).
```

> The contributor then made a copy (fork) of the analysis repository and added their proposed analysis code and results to their fork.

Reviewer: ```suggestion
The contributor then made a copy (fork) of the analysis repository, to which they added their proposed code and results.
```

Medulloblastoma (MB) tumors cluster by molecular subtype, with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (**Figure {@fig:S6}B**), as expected.
Of note, two MB tumors annotated as the SHH subtype did not cluster with the other MB tumors, and one clustered with Group 3 and 4 tumors, suggesting potential subtype misclassification or different underlying biology of these two tumors.
_BRAF_-driven low-grade gliomas (**Figure {@fig:S6}C**) were present in three separate clusters, suggesting that there might be additional shared biology within each cluster.
Histone H3 G35-mutant high-grade gliomas generally clustered together and away from K28-mutant tumors (**Figure {@fig:S6}D**).
Interestingly, although H3 K28-mutant tumors have different biological drivers than do H3 wildtype tumors [@doi:10.1126/science.1232245], they did not form distinct clusters.
This pattern suggests these subtypes may be driven by common transcriptional programs, have other much stronger biological drivers than their known distinct epigenetic drivers, or our sample size is too small to detect transcriptional differences.

> We performed gene set variant analysis (GSVA) for Hallmark cancer gene sets (**Figure {@fig:Fig5}B**) and quantified immune cell fractions using quanTIseq (**Figure {@fig:Fig5}C** and **Figure {@fig:S6}E**), and were able to recapitulate previously-described tumor biology.

Reviewer: ```suggestion
We performed gene set variant analysis (GSVA) for Hallmark cancer gene sets (**Figure {@fig:Fig5}B**) and quantified immune cell fractions using quanTIseq (**Figure {@fig:Fig5}C** and **Figure {@fig:S6}E**), results from which recapitulated previously-described tumor biology.
```

Indeed, we detected significant (Bonferroni-corrected p < 0.05) upregulation of _MYC_ and _E2F_ targets, as well as G2M (cell cycle phase following S phase) in MBs, ATRTs, and HGGs compared to several other cancer groups.
In contrast, LGGs showed significant downregulation (Bonferroni-corrected p < 0.05, multiple cancer group comparisons) of these pathways.
Schwannomas and neurofibromas, which have a documented inflammatory immune microenvironment of T and B lymphocytes as well as tumor-associated macrophages (TAMs), are driven by upregulation of cytokines such as IFN$\gamma$, IL-1, and IL-6, and TNF$\alpha$ [@doi:10.1093/noajnl/vdaa023].
Indeed, we observed significant upregulation of these cytokines in GSVA hallmark pathways (Bonferroni-corrected p < 0.05, multiple cancer group comparisons) (**Figure {@fig:Fig5}B**) and found immune cell types dominated by monocytes in these tumors (**Figure {@fig:Fig5}C**).
We also observed significant upregulation of pro-inflammatory cytokines IFN$\alpha$ and IFN$\gamma$ in both LGGs and craniopharyngiomas when compared to either medulloblastoma or ependymoma tumors (Bonferroni-corrected p < 0.05) (**Figure {@fig:Fig5}B**).

> Together, these results support previous proteogenomic findings of lower immune infiltration in aggressive medulloblastomas and ependymomas versus higher immune infiltration in _BRAF_-driven LGG and craniopharyngiomas [@doi:10.1016/j.cell.2020.10.044].

Reviewer: ```suggestion
Together, these results support previous proteogenomic findings that aggressive medulloblastomas and ependymomas have lower immune infiltration compared to _BRAF_-driven LGGs and craniopharyngiomas [@doi:10.1016/j.cell.2020.10.044].
```
